<?xml version="1.0"?>
<launch>
<group ns="drone08">
  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
  <arg name="name" default="drone08"/>
  <arg name="tf_prefix" value="drone08"/>
  <arg name="model" default="$(find hector_quadrotor_description)/urdf/quadrotor_downward_cam.gazebo.xacro"/>
  <!-- <arg name="x" default="13.0"/>
  <arg name="y" default="13.0"/> 
  <arg name="z" default="0.2"/> -->
  <arg name="x" default="0.0"/>
  <arg name="y" default="-2.0"/> 
  <arg name="z" default="0.3"/>
  </include>
  </group>
  <node pkg="rosservice" type="rosservice" name="enable_motors_d8" args="call --wait /drone08/enable_motors true"/>
</launch>